src/main/java/org/jabref/gui/preferences/GeneralTab.java

```diff
@@ -133,11 +133,16 @@ public Node getBuilder() {
         return builder;
     }
 
+    private void setDisableUpdateTimeStamp() {
+        updateTimeStamp.setDisable(!useTimeStamp.isSelected());
```

Comment:
Well, now I see what you want to achieve. 
You can make use of Javafx binding, that is more elegant.  Under the hood, the bindngs are obversables,
`updateTimpeStamp.disableProperty().bind().not().useTimestamp.selectedProperty() `

This will automatically update the control state based on the property of the other control

Reply:
Much more elegant, I see. Thank you for that suggestion.

While testing this change, I encountered a behavior that struck me as strange: no matter what you _save_ updateTimeStamp as, when the preferences get reloaded, the value of updateTimeStamp is set to match the value of useTimeStamp. In the method `setValues`, around line 141, there are these two lines:

`updateTimeStamp.setSelected(prefs.getBoolean(JabRefPreferences.UPDATE_TIMESTAMP));
    updateTimeStamp.setSelected(useTimeStamp.isSelected());`

This makes me wonder, if updateTimeStamp is always supposed to be the same as useTimeStamp, why there are two different checkboxes. Is this really correct?